README.Rmd

```diff
@@ -106,13 +109,15 @@ ccao::vars_dict %>%
     )
   ) %>%
   mutate(`Unique to Condo Model` = ifelse(
-    var_model_type == "condo" |
-      `Feature Name` %in%
-        c("Condominium Building Year Built", "Condominium % Ownership"),
+    var_name_model != "loc_tax_municipality_name" & (
```

Comment:
A bit hacky here, but without this line the code will decide that `Municipality Name` is unique to the condo model simply because the condo municipality feature is called `loc_tax_municipality_name` while the [equivalent residential model feature](https://github.com/ccao-data/model-res-avm/blob/e956384f966c135d5f2704d139131cdf4645d3f5/params.yaml#L203) is called `loc_cook_municipality_name`. Does this seem reasonable to you?

Reply:
This is actually a mistake, as both should be using `loc_tax_municipality_name` if I recall correctly. We should update the res model feature name to reflect this change. @wrridgeway am I correct here?

Reply:
I can't think of a reason they should be different. I think you're correct.

Reply:
Cool, I opened an issue for the res model fix here! https://github.com/ccao-data/model-res-avm/issues/20